pkg/wfc3/calwf3/calwf3/wf3member.c

```diff
@@ -43,10 +42,9 @@ WF3Info *wf3		o: exposure specific flags and info
 
 	/* Initialize local strings */
 	rootname[0] = '\0';
-	outroot[0]  = '\0';
 	mtype[0]    = '\0';
 
-	strcpy (outroot, asn->product[prodid].subprod[posid].name);
+	const char * outroot = asn->product[prodid].subprod[posid].name;
```

Comment:
The reason for making both ``*_LINE`` & ``*_FNAME`` the same is that the seem to be interchanged throughout the code when they shouldn't be. Having these be identical is the fastest fix (or now).

``FindAsnRoot()`` makes a copy anyhow and then any further persistence use is also copied as it would have to be even if first copied to ``char outroot[SZ_FNAME]``. Furthermore, in this case, there is no further use of ``outname`` beyond ``FindAsnRoot()``. I have to say I'm confused, how exactly is it that you think the thing that ``const char * outname`` points to is going to 'disappear'? If this were true then even the original ``strcpy`` is in jeopardy, regardless of this change.